Runtime/Scripts/LyumaAv3Emulator.cs

```diff
@@ -186,14 +187,18 @@ private void Awake()
 			Camera.onPreCull += PreCull;
 			Camera.onPostRender += PostRender;
 			emulatorInstance = this;
-			ScanForAvatars();
 			if (WorkaroundPlayModeScriptCompile) {
 				LyumaAv3Runtime.ApplyOnEnableWorkaroundDelegate();
 			}
 
 			SceneManager.sceneLoaded += OnSceneLoaded;
```

Comment:
I think this should also move to Start

Reply:
edit: Actually I think this should be removed entirely. I believe Start() will run after all scenes are loaded.